.github/workflows/jest.yml

```diff
@@ -0,0 +1,11 @@
+name: Unit Tests
```

Comment:
I am definitely open to suggestions on the naming of this as well as the name of the file.

Reply:
This works